pkg/filestore/filestore.go

```diff
@@ -172,13 +174,13 @@ func (upload *fileUpload) GetReader(ctx context.Context) (io.ReadCloser, error)
 }
 
 func (upload *fileUpload) Terminate(ctx context.Context) error {
-	if err := os.Remove(upload.infoPath); err != nil {
-		return err
-	}
-	if err := os.Remove(upload.binPath); err != nil {
+	// Account for server crashes or force kills
+	// Say if killed just after removing the bin file
+	err := os.Remove(upload.binPath)
+	if !os.IsNotExist(err) {
```

Comment:
Please use the newer `errors.Is(err, fs.ErrNotExist)` method here.

Reply:
I think it might not work because:
* os.remove wraps the original error `return &PathError{Op: "remove", Path: name, Err: e}`
* os.IsNotExist(err) - retrieves this underlying and matches against `fs.ErrNotExist`.

Reply:
`os.PathError` implements the `Unwrap` method, which is used by `errors.Is` to check if the errors match. The [documentation](https://pkg.go.dev/errors) for `errors` also explicitly mentions `fs.ErrNotExist`, so I would assume it works and that has also been my experience in the past. Or am I missing something?

Reply:
Used `os.ErrNotExist` (which is same as `fs.ErrNotExist`)